### PR TITLE
fix: adding only the proposer in manager initchain

### LINKED ISSUE #603: create validator set only with the proposer
The validator set that is used for the init chain is initialized with the sequencers list obtained from the hub. 
https://github.com/dymensionxyz/dymint/blob/8cd1a1092f93e2d92442978a58f34e00f4b9f4d9/block/manager.go#L134

This validator set should be initialized only with the proposer.

### PR DESCRIPTION
**Pull request by srene**
This PR updates the NewManager function, adding only the proposer as a validator in the init chian

Closes #603 
# PR Standards

## Opening a pull request should be able to meet the following requirements

---

For Author:

- [ ]  Targeted PR against correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to Github issue with discussion and accepted design
- [ ]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

**Comment by mtsitrin** [comment]
@srene I'm handling it already in this PR
https://github.com/dymensionxyz/dymint/pull/606/files#diff-68533b7368c02e034324e64327ea8896e4c292852287d7d2d56aa9b41705cbd0

File: block/manager_test.go [hunk 1]
```diff
@@ -454,3 +454,24 @@ func TestCreateNextDABatchWithBytesLimit(t *testing.T) {
...
+	require.NoError(t, err)
+
+	manager, err := getManager(getManagerConfig(), settlementlc, nil, 1, 1, 0, nil, nil)
+	require.NoError(t, err)
+
+	proposer := settlementlc.GetProposer()
+
+	validatorPubKey := manager.lastState.NextValidators.Validators[0].PubKey
+	require.Equal(t, len(manager.lastState.NextValidators.Validators), 1)
+	require.Equal(t, validatorPubKey.Address(), proposer.PublicKey.Address())
```
Comment (danwt) [inline block/manager_test.go:475]: expected value should come first
require(t, expect, actual)

**Review by danwt — COMMENTED** [review]
it seems @mtsitrin also has a pr so I will pause my review

**Comment by danwt** [comment]
Closing because superseded by #606

**Pull request closed without merging**